cypress/support/fragments/users/userAllFeesFines.js

```diff
@@ -39,6 +47,45 @@ export default {
       });
     });
   },
+  goToOpenFeeFines:() => cy.do(Button({ id: 'open-accounts' }).click()),
+  goToAllFeeFines:() => cy.do(Button({ id: 'all-accounts' }).click()),
   createFeeFine: () => cy.do(Button({ id: 'open-closed-all-charge-button' }).click()),
-  waitLoading:() => cy.expect(HTML('All fees/fines for ').exists())
+  waitLoading:() => cy.expect(HTML('All fees/fines for ').exists()),
+  checkRowsAreChecked:(areRowsChecked) => {
+    const beChecked = areRowsChecked ? 'be.checked' : 'not.be.checked';
+
+    cy.get('#list-accounts-history-view-feesfines')
+      .find('[data-row-index]')
+      .each($row => {
+        cy.get('[class*="mclCell-"]:first-child', { withinSubject: $row }).should(beChecked)
+      })
+  },
```

Comment:
Please describe a reason why you don't use the interactor or add the TODO task to rewrite this piece of code using the interactors

Reply:
To get a specific cell - state of the first cell of each row